v2/rust/Cargo.toml

```diff
@@ -8,3 +8,5 @@ edition = "2021"
 [dependencies]
 anyhow = { version = "*", features = ["backtrace"] }
 clap = { version = "*", features = ["derive"] }
+flexi_logger = "*"
```

Comment:
I think it should do also parallel execution, but we'll have to see that. Otherwise, it's simple to use and offers exactly the options we need (so far).